src/components/StatsInDetail/Charts/OverallTrendsChart.tsx

```diff
@@ -120,6 +120,9 @@ const Chart: React.FC<ChartProps> = ({
                     },
                     {} as Record<string, boolean>
                 ),
+                textStyle: {
+                    color: systemTheme === 'dark' ? '#f0f0f0' : '#777',
+                },
```

Comment:
I am not really sure the issue has to do with the text colors or any css styling. Maybe we will try to create another issue to fix the relative positioning of the numbers on the left-hand side axis, that should address the underlying issue once and for all.

Reply:
> Similar for this. The plugins written here is not visible on the gray background. ![image](https://private-user-images.githubusercontent.com/107131545/384951949-9132c357-3661-4cec-aef7-15ae45876401.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE0Mzk4MzgsIm5iZiI6MTczMTQzOTUzOCwicGF0aCI6Ii8xMDcxMzE1NDUvMzg0OTUxOTQ5LTkxMzJjMzU3LTM2NjEtNGNlYy1hZWY3LTE1YWU0NTg3NjQwMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExMlQxOTI1MzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjQ1YTY2YWNhOWM5OGIyNjY4YTkxMGZkMDFjZWEyMWJiMDRiZGNjNWFjYTE5ZDUxY2Q2NDRjZWY0NGE2NjViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.uWkMC9jf991lEGXen2v1nKZhGtn-nNALdOuEPzp6lqc)



It fixes this. but not really sure if this is the best way to do this

Reply:
My apologies I read too much into the commit message which could be interpreted in more than one way. But I agree this does help with showing the "plugins" label on the chart for the key.

Reply:
Ah, my bad, I was like trying to commit messages in a way that can be easy to filter out whether the issues given here are solved or not and in which commit


Btw can you check if there is something I might have missed out? I tried to fix everything I saw.

Reply:
I think you could double check the new colors in dark mode proposed in this PR matches the ones in the merged PR's I referenced earlier in https://github.com/jenkins-infra/stats.jenkins.io/pull/213#issuecomment-2471304080. 

Other than that we should allow the PR to sit for a few days before checking again to see if we have mised anything. But I think we got most of the relevant features covered. 

Great work @KitsuneKode!

Reply:
I will be resolving the color issue soon. Will do a thorough check.

Thanks. It is one of my first PRs so, I learned a lot.